### ISSUE TITLE
Security Exception when navigating to https URL on Win7

### ISSUE THREAD

**Issue by hrj** [UprootLabs/gngr#10]
As reported by @vit1251 on chat:
```
windows 7 with java version "1.8.0_25"
Java(TM) SE Runtime Environment (build 1.8.0_25-b18)
Java HotSpot(TM) Client VM (build 25.25-b02, mixed mode, sharing)
```
The exception is:
```
An error occurred trying to process a request.
Exception:
java.lang.ExceptionInInitializerError
Meaning:
Unknown.
Message:
null
```

**Comment by hrj** [comment]
Just realized that `"java.lang.RuntimePermission" "loadLibrary.sunmscapi"` needs to be granted to JDK classes. There is a similar grant for `RuntimePermission("loadLibrary.sunec")` already in our security policy.

@vit1251 This should be hopefully fixed in next release.

**Comment by hrj** [comment]
@vit1251 I have published a test-version with a hopeful fix for this issue:
https://uprootlabs.github.io/gngrTestCases/static/gngr_testing_issue10.jar

I don't have a Windows machine available for testing. Can you please download and try this version on your Windows machine?

**Comment by hrj** [comment]
I got my hands on a Win 7 box and was able to reproduce this. Debugging now.

**Comment by hrj** [comment]
The problem turned out to be a very simple one. Windows paths are differently encoded (they are URL escaped) than in *nix.

The previous fix was not required. Hence will be reverting it, and adding a new fix.

**Comment by hrj** [comment]
A new release with a fix is available: [0.3.1](https://github.com/UprootLabs/gngr/releases/tag/v0.3.1)
